### PR TITLE
PrestaShop 8.x support

### PR DESCRIPTION
**Pull request by ZaDarkSide**
Added the minimum amount of changes to allow plugin to work on PrestaShop 8.x tested up to PrestaShop 8.1.3

### CODE REVIEW

**Comment by Fashdey** [comment]
+1 on this

**Comment by eug-L** [comment]
Hi @ZaDarkSide thank you for contributing, tawk.to will actively work towards supporting Prestashop 8. To ensure we can use your code and continue making improvements, we ask that you sign the CLA. Let me know if you have any questions.

**Comment by kreare** [comment]
@ZaDarkSide Please sign the agreement or they can't merge

**Comment by ZaDarkSide** [comment]
@eug-L @kreare CLA is signed

File: prestashop8.x/tawkto.php [hunk 1]
```diff
@@ -120,58 +124,44 @@ public function hookDisplayFooter()
...
+                        if ($options->show_onfrontpage) {
+                            $show = true;
                         }
                     }
                 }
 
-                if (!$show && !in_array($this->context->controller->php_self, ['index', 'category', 'product'])) {
-                    return;
+                if (!$show) {
+                    return '';
```
Comment (eug-L) [inline prestashop8.x/tawkto.php:152]: check if return should be void or string

File: prestashop8.x/upgrade/Upgrade-1.2.0.php [hunk 2]
```diff
@@ -1,139 +0,0 @@
-<?php
```
Comment (eug-L) [inline prestashop8.x/upgrade/Upgrade-1.2.0.php]: restore upgrades, and add upgrade script for 1.3.0

Comment (eug-L) [reply prestashop8.x/upgrade/Upgrade-1.2.0.php]: check upgrade flow

prestashop 7 to 8, then upgrade module

File: prestashop8.x/tawkto.php (hunk 1 above)
Comment (eug-L) [reply prestashop8.x/tawkto.php]: standardized to void

**Pull request merged**